apps/console/src/features/console-settings/components/console-roles/create-console-role-wizard/create-console-role-wizard-basic-info-form.tsx

```diff
@@ -63,11 +53,13 @@ const CreateConsoleRoleWizardBasicInfoForm: FunctionComponent<CreateConsoleRoleW
             helperText={ (
                 <Hint>
                     Provide a distinctive and meaningful display name for the role. This name should clearly represent
-                    the purpose or responsibilities associated with this role within the Console application. E.g.{ " " }
-                    <Code>application:read</Code>
+                    the purpose or responsibilities associated with this role within the Console application.
+                    E.g.{ " " }<Code>application:read</Code>
                 </Hint>
             ) }
             placeholder="Enter Role name"
+            maxLength={ 10 }
```

Comment:
Add this to a constant file.

Reply:
These validation props are not required here. Therefore removed with 2f40fd5

Reply:
Ack